src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -801,6 +792,42 @@ public boolean isKnownParticipant( UUID studyId, String participantId ) {
                 .collect( Collectors.toSet() );
     }
 
+    @Scheduled( fixedRate = 60000 )
+    public void refreshUserAppsDictionary() {
+        SearchApi searchApi;
+        try {
+            ApiClient apiClient = apiClientCache.get( ApiClient.class );
+            searchApi = apiClient.getSearchApi();
+        } catch ( ExecutionException e ) {
+            logger.error( "Unable to load api" );
+            return;
+        }
+
+        logger.info( "Refreshing chronicle user apps dictionary" );
+
+        List<Map<FullQualifiedName, Set<Object>>> searchResult = searchApi
+                .executeEntitySetDataQuery( userAppsDictESID, new SearchTerm( "*", 0, SearchApi.MAX_SEARCH_RESULTS ) )
+                .getHits();
```

Comment:
You can just get the entire entity set with `DataApi.loadEntitySetData`.

https://github.com/openlattice/api/blob/develop/src/main/java/com/openlattice/data/DataApi.java#L74